package.json

```diff
@@ -1,5 +1,12 @@
 {
   "name": "react-numerics",
+  "main": "./dist/react-numerics.umd",
+  "types": "./dist/index.d.ts",
+  "files": [
+    "dist/",
+    "libs/react-numerics/src",
+    "!libs/react-numerics/src/**/*.test.ts"
```

Comment:
you can ignore stories in `files` and the tsconfig as well so they don't get put into the build